src/software/ai/passing/pass_generator.h

```diff
@@ -128,6 +133,9 @@ namespace Passing
         // (pass_start_x, pass_start_y, pass_speed, pass_start_time)
         static const int NUM_PARAMS_TO_OPTIMIZE = 4;
 
+        // The number of iterations to run on each call to `getBestPassSoFar()`
+        // **if** running determinstically
+        static const size_t NUM_ITERS_PER_DETERMINISTIC_CALL = 10;
```

Comment:
Where did this number come from? The average number of iterations run while using threads?

Reply:
Fairly random to be honest, just made sure it worked well for tests. Eventually this will probably need to be exposed, but in the interests of keeping this small, YAGNI.

Reply:
Can you leave a quick comment saying that? Just to give people enough context that this _isn't_ a carefully chosen value based on a bunch of tests with threading.

Reply:
Yep, no problem. Added a comment.